.github/workflows/docs-ci.yml

```diff
@@ -140,6 +140,14 @@ jobs:
           remote_user: ${{ secrets.DOCS_USER }}
           remote_key: ${{ secrets.DEEPHAVEN_CORE_SSH_KEY }}
 
+      - name: Upload JVM Error Logs
+        uses: actions/upload-artifact@v3
+        if: failure()
```

Comment:
Does this need to be in double `${{ }}` ? Like https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-status-check-functions

Reply:
This example was essentially copied from other github examples. Maybe they recommend always using `${{ ... }}` now, but it _does_ work without. If we want to update the syntax, we should probably do it everywhere.

https://docs.github.com/en/actions/learn-github-actions/expressions#about-expressions
> When you use expressions in an if conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression.